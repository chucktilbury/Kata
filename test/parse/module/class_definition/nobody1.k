;-------------------------------------------------------------------------------
; Verify no body with filled parent
;-------------------------------------------------------------------------------

class name (compound.name)
