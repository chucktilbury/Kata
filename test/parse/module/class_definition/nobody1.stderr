Syntax: /home/chuck/Src/Kata/test/parse/module/class_definition/nobody1.k: 6: 1: expected a '{' but got end of file
Syntax: /home/chuck/Src/Kata/test/parse/module/class_definition/nobody1.k: 6: 1: expected module item or end of file but got class
Syntax: /home/chuck/Src/Kata/test/parse/module/class_definition/nobody1.k: 6: 1: module cannot be empty
